config.js

```diff
@@ -458,64 +422,71 @@ exports.updateFromArgv = function() {
+    const deprecatedArgv = [
+         [["botid", "bot", "id"], "username"],
+         [["minrankedhandicap"], "minhandicapranked"],
+         [["minunrankedhandicap"], "minhandicapunranked"],
+         [["maxrankedhandicap"], "maxhandicapranked"],
+         [["maxunrankedhandicap"], "maxhandicapunranked"],
+         [["maxtotalgames"], "maxconnectedgames"],
+         [["maxactivegames"], "maxconnectedgamesperuser"],
+         [["maxmaintime"],  getBLCDeprecationString("maxmaintime", "")],
+         [["maxmaintimeranked"], getBLCDeprecationString("maxmaintime", "ranked")],
+         [["maxmaintimeunranked"], getBLCDeprecationString("maxmaintime", "unranked")],
+         [["minmaintime"], getBLCDeprecationString("minmaintime", "")],
+         [["minmaintimeranked"], getBLCDeprecationString("minmaintime", "ranked")],
+         [["minmaintimeunranked"], getBLCDeprecationString("minmaintime", "unranked")],
+         [["maxperiodtime"], getBLCDeprecationString("maxperiodtime", "")],
+         [["maxperiodtimeranked"], getBLCDeprecationString("maxperiodtime", "ranked")],
+         [["maxperiodtimeunranked"], getBLCDeprecationString("maxperiodtime", "unranked")],
+         [["minperiodtime"], getBLCDeprecationString("minperiodtime", "")],
+         [["minperiodtimeranked"], getBLCDeprecationString("minperiodtime", "ranked")],
+         [["minperiodtimeunranked"], getBLCDeprecationString("minperiodtime", "unranked")],
+         [["maxperiods"],  getBLCDeprecationString("maxperiods", "")],
+         [["maxperiodsranked"], getBLCDeprecationString("maxperiods", "ranked")],
+         [["maxperiodsunranked"], getBLCDeprecationString("maxperiods", "unranked")],
+         [["minperiods"], getBLCDeprecationString("minperiods", "")],
+         [["minperiodsranked"], getBLCDeprecationString("minperiods", "ranked")],
+         [["minperiodsunranked"], getBLCDeprecationString("minperiods", "unranked")],
+         [["ban"], "bans"],
+         [["banranked"], "bansranked"],
+         [["banunranked"], "bansunranked"],
+         [["boardsize"], "boardsizes"],
+         [["boardsizeranked"], "boardsizesranked"],
+         [["boardsizeunranked"], "boardsizesunranked"],
+         [["boardsizewidths", "boardsizewidthsranked", "boardsizewidthsunranked",
+           "boardsizeheights", "boardsizeheightsranked", "boardsizeheightsunranked"], "boardsizes"],
+         [["komi"], "komis"],
+         [["komiranked"], "komisranked"],
+         [["komiunranked"], "komisunranked"],
+         [["speed"], "speeds"],
+         [["speedranked"], "speedsranked"],
+         [["speedunranked"], "speedsunranked"],
+         [["timecontrol"], "timecontrols"],
+         [["timecontrolranked"], "timecontrolsranked"],
+         [["timecontrolunranked"], "timecontrolsunranked"]
+    ];
+    for (const [oldNames, newName] of deprecatedArgv) {
+        for (const oldName of oldNames) {
+            if (argv[oldName]) {
+                console.log(`Deprecated: --${oldName} is no longer `
```

Comment:
`Deprecated` usually means that a feature will be removed in future released, we should probably not use that word unless we go around and fix the config to use the new value (until we actually drop support sometime in the future?)

Anyway I'm ok with not supporting these arguments, but if we do we should not use the word "deprecated", instead stick to just "unsupported/no longer supported" etc.

Reply:
aaaah important nuance!!!!

so `deprecated` refers to code that is still existing and working but not supported anymore!

Reply:
Actually it means: Still supported right now, but support will be dropped in the future.
That does usually mean no "support" support, like no fixes to this broken feature. But also means current code that uses this should keep working.